oss-internship-2020/libuv/generator/wrapper_generator.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2020 Google LLC
```

Comment:
Can you add Python3 type annotations?

See https://google.github.io/styleguide/pyguide.html#221-type-annotated-code